ceph_deploy/osd.py

```diff
@@ -379,7 +379,7 @@ def activate(args, cfg):
         time.sleep(5)
         catch_osd_errors(distro.conn, distro.conn.logger, args)
 
-        if distro.is_el:
+        if distro.is_el and distro.init != 'systemd':
             system.enable_service(distro.conn)
```

Comment:
or rather, i think it should be if systemd (enable ceph.target) else if sysvinit (enable ceph) .. it has nothing to do with is_el, right?